demisto_sdk/commands/common/hook_validations/image.py

```diff
@@ -158,3 +160,15 @@ def is_not_default_image(self):
                 self._is_valid = False
                 return False
         return True
+
+    def is_valid_image_name(self):
+        """Check if the image name is valid"""
+
+        if not self.file_path.endswith("_image.png") and not self.file_path.endswith("_Image.png"):
```

Comment:
```suggestion
        if not self.file_path.endswith("_image.png"):
```
we should only support `_image`

Reply:
So we'll have to rename more integration images to pass them validations.

Reply:
done.
https://github.com/demisto/content/pull/12416